themes/classic/_dev/css/partials/_variables.scss

```diff
@@ -5,6 +5,7 @@ $gray-light: #F1F1F1;
 $gray-light-second: #DBDBDB;
 $gray-lighter: #f6f6f6;
 $gray-lighter-second: #F7F7F7;
+$black: #000;
```

Comment:
Is this really needed?

Reply:
No problem, I can remove it from variables files if needed